js/packages/web/src/views/artCreate/infoStep.tsx

```diff
@@ -121,6 +121,16 @@ export const InfoStep = (props: {
 
               <label>
                 <h3>Maximum Supply</h3>
+                <p
+                  style={{
+                    opacity: '0.85',
+                    margin: '-4px 0 0',
+                    paddingBottom: '10px',
+                  }}
+                >
+                  <strong>NOTE:</strong> If this field is left blank, it will be
+                  possible to mint an unlimited number of editions of this NFT.
+                </p>
                 <Form.Item name={['properties', 'maxSupply']}>
                   <InputNumber
```

Comment:
Maybe we do something with the placeholder.

```
Quantity (Leave blank for unlimited  and set to 0 for 1/1)
```



And if you are up for it ideal is to replace this with a button group

(One of One / Limited Edition / Unlimited)

Default to 1/1 which sets maxSupply to 0.
Selecting Limited edition reveals input field for entering a supply that is greater than 0 (should be 1 but some people do like generating the edition record).
Unlimited drops the maxSupply from properties.